ShiftsLogger.StanimalTheMan/ShiftsLogger.StanimalTheMan/ShiftsLoggerWebAPI.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
Weird that the ShiftsLoggerWebAPI is the name of the project on my end locally but is outdated on GitHub as I renamed it to better differentiate the Console UI from the WebAPI after the initial web api project creation.